.github/workflows/docs.yml

```
@@ -0,0 +1,64 @@
+name: Docs
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
```

won't pr builds in this case kill a running build on main branch? what about:
```suggestion
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
```

---

@jezdez - this comes from the conda-libmamba-solver docs. Any comments?

---

Ignore this, the workflow in conda-libmamba-solver does not run on PRs.

The idea is that if we merge too many PRs at once, subsequent pushes to main do not run into deployment run conditions and only the last one is pushed.

So we need a ternary conditional like:

`<your suggested expression> if on_pull_request else "pages"`

Which I _think_ can be done like this:

```suggestion
  group: ${{ github.event_name == 'pull_request' && join([github.workflow, (github.event.pull_request.number || github.sha)], '-') || 'pages' }}
 ```

---

@dbast - wdyt?

---

can be simplified to:
`pages-${{ github.workflow }}-${{ github.event.pull_request.number }}`  no && and || required as values will afaik be just empty in case of not a PR.